java-squid/src/test/files/se/ConditionAlwaysTrueOrFalseCheck.java

```diff
@@ -1250,4 +1250,23 @@ private void orEqualAssignement(boolean a) {
     foo &= a;
     if(foo) {} //false negative: Should be fixed by SONARJAVA-1391
   }
+ 
+  public static void awaitUninterruptibly(CountDownLatch latch) {
```

Comment:
Why this pull request about unclosed resources modifies this test file of another check ?